stdlib/effect.mli

```diff
@@ -16,13 +16,21 @@
 
     @since 5.0 *)
 
-type _ t = ..
+type 'a t = 'a eff = ..
 (** The type of effects. *)
 
+exception Unhandled_effect : 'a eff -> exn
+(** Exception raised when an effect is performed and there is no matching
+    handler for that effect. The argument is the unhandled effect. *)
+
+exception Continuation_already_taken
+(** Exception raised when a continuation is resumed with a [continue] or
+    [discontinue] but the continuation has already been resumed. *)
+
 external perform : 'a t -> 'a = "%perform"
 (** [perform e] performs an effect [e].
 
-    @raise Unhandled if there is no active handler. *)
+    @raise Unhandled_effect if there is no active handler. *)
```

Comment:
Is the concept of *active* handler defined somewhere (e.g. in supporting docs, sorry I hadn't the time to sift through them yet) ? 

If not I don't like the term, it refers to something you can deactivate but AFAIK we are rather looking a syntactic property of programs (presence of a surrounding handler). Maybe simply:

```
@raise Unhandled e if there is no handler for [e].
```